packages/cli/src/util/get-latest-version/get-latest-worker.js

```diff
@@ -247,6 +247,14 @@ async function fetchDistTags(name) {
         });
         res.on('end', () => {
           try {
+            if (res.statusCode && res.statusCode >= 400) {
+              return reject(
```

Comment:
Don't think you need the return here

Reply:
This `return` will prevent the rejected promise from attempting to resolve/reject later. To my knowledge, `reject()` does not stop the execution of the current scope.

Reply:
You're correct, it doesn't stop the execution, but I don't believe you can resolve after a promise rejects

Reply:
I'd rather not allow further execution that happens to not do anything. I think it's more correct and less brittle to return here.